modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -54,15 +54,20 @@ This setting allows {productname} to automatically pick the latest distribution
 If you add new distributions later, {productname} will use the most recently created or modified.
 ====
 
-You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
-
 [NOTE]
 ====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
 ====
 
+You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking on the name of the profile you want to edit.
+
+To adjust advanced {kickstart} configuration options, navigate to the menu:Autoinstallation Details[Advanced Options] subtab.
```

Comment:
This page was renamed to Advanced Autoinstallation Configuration. See the after screenshot in the PR

Reply:
Good spot :+1: